vllm/engine/metrics.py

```diff
@@ -600,10 +638,39 @@ def _log_prometheus(self, stats: Stats) -> None:
                             stats.time_decode_requests)
         self._log_histogram(self.metrics.histogram_time_in_queue_request,
                             stats.time_in_queue_requests)
+        self._log_histogram(
```

Comment:
The way this is computed, it looks like a duplicate of:
- `vllm:time_to_first_token_seconds`